Makefile

```diff
@@ -26,7 +26,7 @@ else
 # the rest of the Makefile is only visible to Cargo
 
 cargo-prep: $(OUT_DIR)/config.rs
-	@echo "cargo:rustc-flags=-l $(LUA_LIBNAME) -L native=$(LUA_LIBDIRS)"
+	@echo "cargo:rustc-flags=-l $(LUA_LIBNAME) -L $(LUA_LIBDIRS)"
```

Comment:
Oops, i don't have experience with git...
I thought when i make the pull request if i edit my fork you'll not see :/
I just remove "native=" because cargo don't want it.